mapper/pojo/src/test/java/org/hibernate/search/mapper/pojo/dirtiness/building/impl/PojoAssociationPathInverterTest.java

```diff
@@ -189,6 +189,7 @@ public void detectInfiniteRecursion() {
 
 	private PropertyHandle setupPropertyStub(PojoTypeModel<?> holdingTypeMock, String propertyName,
 			PojoGenericTypeModel<?> propertyTypeMock) {
+		// It seems that we cannot add a generic type such a <? super P> here
```

Comment:
Just use `PropertyHandle<?>` here, and cast it to `(PropertyHandle)` where you get the compilation error.
That's how we have to deal with generics when using mocks, unfortunately. You can find similar examples all over the place.

Also, please add the missing `<?>` at line 72 (there's a raw `PropertyHandle`).